src/store/modules/settings.js

```diff
@@ -36,85 +36,85 @@ export default {
       tabSize: 2,
     },
     loaded: false,
-    theme: 'dark',
+    theme: "auto",
```

Comment:
Since this file only has a 1-line change, would you mind resetting it and just committing this 1 line? You should be able to reset the file with this command and then you can add this 1-line change back.

```bash
git checkout main -- src/store/modules/settings.js
```